index.ftd

```diff
@@ -1,51 +1,1944 @@
+copyright: $footer.copyright
+logo-height: $footer.logo-height
+background-color: $footer.background-color
+footer-list: $footer.footer-list
+logo-1: $footer.logo-1
+logo-2: $footer.logo-2
+logo-3: $footer.logo-3
+site-url: $footer.site-url
+site-logo: $footer.site-logo
+social: $footer.social
+
+-- end: ftd.desktop
+
+-- ftd.mobile:
+
+-- footer-mobile:
+site-name: $footer.site-name
+telegram-url: $footer.telegram-url
+instagram-url: $footer.instagram-url
+twitter-url: $footer.twitter-url
+discord-url: $footer.discord-url
+facebook-url: $footer.facebook-url
+logo-5: $footer.logo-5
+logo-4: $footer.logo-4
+bio: $footer.bio
+copyright: $footer.copyright
+logo-height: $footer.logo-height
+background-color: $footer.background-color
+footer-list: $footer.footer-list
+logo-1: $footer.logo-1
+logo-2: $footer.logo-2
+logo-3: $footer.logo-3
+site-url: $footer.site-url
+site-logo: $footer.site-logo
+social: $footer.social
+
+-- end: ftd.mobile
+
+-- end: ftd.column
+
+-- end: footer
+
+
+
+
+
+
+
+
+
+
+-- component footer-desktop:
+optional string site-name:
+optional string site-url:
+optional ftd.image-src logo-1: 
+optional ftd.image-src logo-2:
+optional ftd.image-src logo-3: 
+optional ftd.image-src logo-4:
+optional ftd.image-src logo-5:
+optional string twitter-url: /
+optional string facebook-url: /
+optional string discord-url: /
+optional string instagram-url:/ 
+optional string telegram-url: /
+optional string copyright:
+optional body bio:
+optional ftd.image-src site-logo:
+integer logo-height:
+boolean social:
+optional ftd.color background-color:
+pr.toc-item list footer-list:
 
 -- ftd.column:
+width: fill-container
+padding-vertical.px: 40
+background.solid: $footer-desktop.background-color
+spacing.fixed.px: 24
+
+-- ftd.row:
+width: fill-container
+spacing.fixed.px: 40
+padding-horizontal.px: 40
+
+-- logo-with-name:
+site-name: $footer-desktop.site-name
+site-url: $footer-desktop.site-url
+logo-height: $footer-desktop.logo-height
+
+-- ftd.text:
+if: { footer-desktop.bio != NULL }
+text: $footer-desktop.bio
+role: $inherited.types.copy-small
+color: $inherited.colors.text
+width.fixed.px: 305
 align-self: center
 
--- chat.message-left: Yo! I am Amit Upadhyay, CEO of FifthTry, welcome to the new world of FTD.
+-- end: ftd.row
+
+-- ftd.row:
+width: fill-container
+border-bottom-width.px: 1
+border-color: $inherited.colors.border
+
+-- end: ftd.row
+
+-- ftd.row:
+padding-horizontal.px: 40
+width: fill-container
+align-content: center
+
+-- ftd.text: $footer-desktop.copyright
+color: $inherited.colors.text
+role: $inherited.types.label-small
+width: fill-container
+
+-- ftd.row:
+if: { !footer-desktop.social }
+align-content: center
+spacing.fixed.px: 32
+
+-- footer-list-section: 
+title: $obj.title
+url: $obj.url
+is-active: $obj.is-active
+sub-sections: $common.empty-toc
+$loop$: $footer-desktop.footer-list as $obj
+
+-- end: ftd.row
+
+-- ftd.row:
+spacing.fixed.px: 32
+if: { footer-desktop.social }
+
+-- images-list:
```

Comment:
@saurabh-lohiya footer component should be like:
https://github.com/fastn-community/set-1-ds/blob/8c2335a217c3a4fa37f829cd4280307baccf51ee/index.ftd#L576